scripts/install.sh

```diff
@@ -739,6 +739,18 @@ if [ $DIRECTORY != "-" ]; then
   fi;
 fi;
 
+if "$DIRECTORY"/bin/octoprint config get --yaml "api.allowCrossOrigin" | grep -q 'false'; then
+yes_no=( 'yes' 'no' )
+
+list_input ${red}${bold}"Should I enable CORS ? FYI, this is required by OctoDash v3 and OctoPrint 1.6.0, and may have security implications"${normal} yes_no cors
```

Comment:
```suggestion
list_input "Should I enable CORS ? FYI, this is required by OctoDash v3 and OctoPrint 1.6.0, and may have security implications"${normal} yes_no cors
```

I would prefer this not being red, as this is nothing bad (yet).